Editorial clean up of new vector index page (#695)
neo4j/docs-cypher: modules/ROOT/pages/indexes-for-vector-search.adoc

:l2-norm: image:l2.svg["l2"]-norm

[role=beta]
[[indexes-vector]]
= Vector search index

> _Vector search indexes was released as a public beta in Neo4j 5.11._

Reviewer: ```suggestion
_Vector search indexes were released as a public beta in Neo4j 5.11._
```

Reviewer: I know you mean that it's a feature but it looks a bit off.

|===

> The results are expected, with papers discussing graph-based nearest-neighbor searches.

Reviewer 2: Though perhaps in need of a reword or thought; it would be nice to still point out that the results indeed are in a similar "space" to query vector. Thus the vector search (k-ANN) is working correctly as intended.

Author: Ok I will add it back in for now, and think about a rewording later :)

[[indexes-vector-similarity]]
== Supported similarity functions

> The choice of similarity function can vastly differ depending on which indexed vectors are considered similar, and which are valid.

Reviewer 2: This still reads the wrong way around to me.
So I think we can change the line to something like:
```suggestion
The choice of similarity function affects which indexed vectors are considered similar, and which are valid.
```

A valid vector for a Euclidean vector index is when all vector components can be represented finitely in {ieee-754} single precision.

Euclidean interprets the vectors in Cartesian coordinates.

> The measure is related to the Euclidean distance, i.e. how far two points are from one another.

Reviewer 2: I'm aware that `i.e.` _without_ the trailing comma is the British English style, and _with_ the trailing comma (`i.e.,`) is the American English style.

I was under the impression we (unfortunately) use the American conventions; therefore, should we have the trailing comma, or use some English connector instead?

Author: I will add the comma (and strong agree on the preferred use of convention)
